MetaMask/metamask-mobile
# Fix approval transaction getting mistakenly treated as a swap transaction

**Description**

This PR fixes approval transaction getting mistakenly treated as a swap transaction.

How to test:
Go to https://app.olympusdao.finance/#/bonds/eth and try to bond wETH, if a transaction modal appears then it's because it is working correctly.
